File: FedRaft-Core/src/test/org/bupt/cad/fedraft/beans/NodeInfoTest.java
package org.bupt.cad.fedraft.beans;

import org.junit.Test;

import java.net.UnknownHostException;

public class NodeInfoTest {

    @Test
    public void modTest() throws UnknownHostException {
        NodeInfo node = new NodeInfo("127.0.0.1", 16777, 16799);
        long id = node.getNodeId();
        NodeInfo nodeInfo = new NodeInfo(id);
        System.out.println(nodeInfo);
    }

}

File: FedRaft-ModelTrain/test/NodeInfoTest.py
from beans.NodeInfo import NodeInfo
       

def test():
    nodeinfo = NodeInfo("10.112.195.22", 16555, 16789)
    print(f"nodeinfo = {nodeinfo.get_id()}")
    print(f"nodeinfoID = {NodeInfo.id_to_ip(nodeinfo.get_id())}")
                 